### PR TITLE
Ramp tutorial modeling

### PR DESCRIPTION
**Pull request by patrickmoc**
- [Added modeling ramp section in the tutorial](https://drive.google.com/file/d/1GsMBNOItsnpf3v8W_VZJeMTPjWLXJR2w/view?usp=sharing)
- [Added note in limitations of precision section](https://drive.google.com/file/d/1rDUwllEgK3PrFi__OqzcEoN_64XSqfiu/view?usp=sharing)
- Added helpful links related to modeling ramps

### CODE REVIEW

File: _posts/2019-09-15-guide_create_surf_ramps.md [hunk 1]
```diff
@@ -319,12 +328,32 @@ The vmf created during this guide is available [here](/assets/mapsrc/surf_ramp_t
...
+  
+The use of modeled ramps can also help avoid reaching hammer's brush limits by shifting the majority of the detail to static props.
+  
+Two common tools used to create these models are <a href="https://github.com/tuxxi/propper-2013" target="_blank">Propper</a> and <a href="http://steamreview.org/BlenderSourceTools/" target="_blank">Blender Source Tools</a>.
+  
+Propper is an addon for hammer that allows you to convert world brushes directly into compiled model files. 
+  
+Blender Source Tools is an addon for Blender that allows you to export your blender models as source model files.
+  
+Creating and texturing models is a massive topic on it's own so this guide will not look in depth at creating these models.
```
Comment (Gocnak) [inline _posts/2019-09-15-guide_create_surf_ramps.md]: its*

File: _posts/2019-09-15-guide_create_surf_ramps.md [hunk 2]
```diff
@@ -123,11 +123,20 @@ For curved ramps this means that the portion near the bottom of the ramp where s
 The standard fix for this has been to add a flat "base" portion at the bottom of the ramp segment that you can't surf on. 
 This means that any precision errors would only take place at the base of the ramp and not the surfable portion.  
   
+This is explained in more depth <a href="https://youtu.be/jcC2UKydb9Q" target="_blank">here</a>.
```
Comment (Gocnak) [inline _posts/2019-09-15-guide_create_surf_ramps.md]: Any reason to not use the markdown style for links? HTML is great but markdown is much nicer to read. https://guides.github.com/features/mastering-markdown/

HTML should really only be used when necessary (custom styling). Markdown is pretty powerful for a lot of use cases.

File: _posts/2019-09-15-guide_create_surf_ramps.md [hunk 3]
```diff
@@ -319,12 +328,32 @@ The vmf created during this guide is available [here](/assets/mapsrc/surf_ramp_t
...
 ## Modeling Ramps
+<img src="/assets/images/ramps_guide/headers/nebula.jpg" alt="Model Header" style="display: block; margin: auto">
+
+While a lot can be accomplished with brushes, complex brushes and ramps will sometimes reach the limits of what can be done with them. 
+At this point it may be time to look into making the ramps into models.  
+  
+Normally this doesn't need to be done until late in development when doing an art pass for details, 
+but it tends to be a lot more involved and requires experience with modeling software.  
+  
+Model collision can also be prone to having far more rampbugs.
```
Comment (Gocnak) [inline _posts/2019-09-15-guide_create_surf_ramps.md]: Model collision without clipping right?

File: _posts/2019-09-15-guide_create_surf_ramps.md (hunk 1 above)
Comment (patrickmoc) [reply _posts/2019-09-15-guide_create_surf_ramps.md]: Fixed in commit [
f0f6c68](https://github.com/momentum-mod/docs/pull/20/commits/f0f6c681c2889d14c7fa5a39388a72da1a22983c)

File: _posts/2019-09-15-guide_create_surf_ramps.md (hunk 3 above)
Comment (patrickmoc) [reply _posts/2019-09-15-guide_create_surf_ramps.md]: Added clarification in commit [
24b47d2](https://github.com/momentum-mod/docs/pull/20/commits/24b47d20defd12aca1bfbf29cadd92753ae1d9a9)

File: _posts/2019-09-15-guide_create_surf_ramps.md (hunk 2 above)
Comment (patrickmoc) [reply _posts/2019-09-15-guide_create_surf_ramps.md]: The original idea was to have the external links open in a new tab, but there is a way to do that through markdown so i will remove the html

Comment (patrickmoc) [reply _posts/2019-09-15-guide_create_surf_ramps.md]: fixed in commit [dab0b8c](https://github.com/momentum-mod/docs/pull/20/commits/dab0b8ccdd73b788cb638b3e8e4ec5e020035fc0)

**Pull request merged**